tests/components/LocationBias.test.tsx

```diff
@@ -68,8 +63,8 @@ beforeEach(() => {
 
 it('renders the proper text (location DisplayName, method, and update btn)', () => {
   render(<LocationBias />);
-  const expectedLocationName = mockedStateVaDevice.location.locationBias.displayName;
-  const locationNameElement = screen.getByText(expectedLocationName);
+  const expectedLocationName = mockedStateVaDevice.location?.locationBias?.displayName;
+  const locationNameElement = expectedLocationName && screen.getByText(expectedLocationName);
```

Comment:
This is needed because of the `Partial<State>` type for `mockedStateVaDevice` right? Since these fields are technically defined in the const variable, I think it would be best if we don't need to do checks like this in multiple places.

What do you think about updating the type for `mockedStateVaDevice` to something like `Partial<State & { location: LocationBias }>` and we can keep these two lines as before, without the optional chaining and conditional guards? If so, we can update the types for other const variables for mocking states as well

Reply:
sure, updated the type so the optional chaining and guards could be removed